src/Suave/Owin.fs

```diff
@@ -0,0 +1,546 @@
+﻿module Suave.Owin
+
+// Following the specification:
+// https://github.com/owin/owin/blob/master/spec/owin-1.1.0.md
+
+open System
+open System.Net
+open System.Threading
+open System.Collections
+open System.Collections.Generic
+open System.Threading.Tasks
+open System.Security.Claims
+open System.Security.Principal
+open System.Runtime.InteropServices
+open Suave
+open Suave.Http
+open Suave.Types
+open Suave.Sockets
+
+type OwinEnvironment =
+  IDictionary<string, obj>
+
+type OwinApp =
+  OwinEnvironment -> Async<unit>
+
+type OwinAppFunc =
+  Func<OwinEnvironment, Task>
+
+[<CompilationRepresentation(CompilationRepresentationFlags.ModuleSuffix)>]
+module OwinConstants =
```

Comment:
Is it worth potentially pulling in (either as a dependency or source file via paket) the `Constants` definitions from Freya.Core?